### PR TITLE
polish: add cron to wrangler.toml when new Scheduled Worker is created

### PR DESCRIPTION
**Pull request by timabb031**
Enhances #1113 / relates to #1175

### CODE REVIEW

File: packages/wrangler/src/index.tsx [hunk 1]
```diff
@@ -628,21 +628,23 @@ function createCLIParser(argv: string[]) {
...
-          // rewrite wrangler.toml with main = "path/to/script"
+          // rewrite wrangler.toml with main = "path/to/script" and any additional config specified in `extraToml`
           const parsedWranglerToml = parseTOML(
             readFileSync(wranglerTomlDestination)
           );
-          fs.writeFileSync(
-            wranglerTomlDestination,
-            TOML.stringify({
+          const newToml = {
+            ...{
```
Comment (petebacondarwin) [inline packages/wrangler/src/index.tsx:640]: I don't think you need to spread over these properties...

```ts
const newToml = {
  name: parsedWranglerToml.name,
  main: scriptPath,
  compatibility_date: parsedWranglerToml.compatibility_date,
  ...extraToml,
};

File: packages/wrangler/templates/new-worker-scheduled.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * - Run `wrangler dev --local` in your terminal to start a development server
  * - Run `curl "http://localhost:8787/cdn-cgi/mf/scheduled"` to trigger the scheduled event
  * - Go back to the console to see what your worker has logged
- * - Add a Cron trigger to wrangler.toml (see https://developers.cloudflare.com/workers/wrangler/configuration/#triggers)
+ * - Update the Cron trigger in wrangler.toml
```
Comment (petebacondarwin) [inline packages/wrangler/templates/new-worker-scheduled.js:7]: Out of curiosity, why remove the URL here? I thought that was quite useful?

**Review by petebacondarwin — COMMENTED** [review]
This is great. Could you add some unit tests?

File: packages/wrangler/src/index.tsx (hunk 1 above)
Comment (timabb031) [reply packages/wrangler/src/index.tsx:640]: Ok, I'll update it as your comment.

File: packages/wrangler/templates/new-worker-scheduled.js (hunk 2 above)
Comment (timabb031) [reply packages/wrangler/templates/new-worker-scheduled.js:7]: The example cron trigger in the docs link is exactly the same as what this PR adds to wrangler.toml, so I thought I'd remove it.  I can add it it back in though.

**Comment by timabb031** [comment]
> This is great. Could you add some unit tests?

Yes ok, I'll aim to add some tests this week at some point.

**Review by petebacondarwin — APPROVED** [review]
Marvellous! Thanks @timabb031 !

**Pull request merged**